src/Mealz/AccountingBundle/Tests/Controller/AccountingBookControllerTest.php

```diff
@@ -197,9 +197,10 @@ protected function getRawResponseCrawler()
 
     /**
      * Tests if finance staff can access the transaction export page and admins and default users can not
+     *
      * @test
```

Comment:
@test is redundant

Reply:
I got 57 matches should I change them all?

Reply:
Removed all :-) Add prefix "test" to all method names.